samples/msal-node-samples/ElectronSystemBrowserApp/src/config/customConfig.json

```
@@ -0,0 +1,29 @@
+{
+    "authOptions":
+        {
+            "clientId": "8fcb9fc1-d8f9-49c0-b80e-a8a8a201d051",
```

```suggestion
            "clientId": "ENTER_CLIENT_SECRET",
```
Can we update the values in this file with instructions instead of defaults?

---

All the electron samples follow the exact implementation. Should I update them all?

---

Right, I think `customConfig.json` should have default values (folks end up trying to sign-in with them), but let's leave the others to another PR.